src/test/resources/fixtures/1.30/distribution.json

```diff
@@ -0,0 +1,10 @@
+
+  "Descriptor": {
+    "MediaType": "application/vnd.docker.distribution.manifest.v2+json",
+    "Digest": "sha256:c0537ff6a5218ef531ece93d4984efc99bbf3f7497c0a7726c88e2bb7584dc96",
+    "Size": 3987495,
+    "URLs": [
+      ""
+    ]
+  }
```

Comment:
Can you change to this?

```
{
  "Descriptor": {
    "MediaType": "application/vnd.docker.distribution.manifest.v2+json",
    "Digest": "sha256:c0537ff6a5218ef531ece93d4984efc99bbf3f7497c0a7726c88e2bb7584dc96",
    "Size": 3987495,
    "URLs": [
      "url1",
      "url2"
    ]
  },
  "Platforms": [
    {
      "Architecture": "amd64",
      "OS": "linux",
      "OSVersion": "",
      "OSFeatures": [
        "feature1",
        "feature2"
      ],
      "Variant": "",
      "Features": [
        "feature1",
        "feature2"
      ]
    }
  ]
}
```

Reply:
Done it, also fixed the test that references this file.